CRM/Utils/Check/Component/Env.php

```diff
@@ -1097,6 +1097,31 @@ public function checkMysqlVersion() {
     return $messages;
   }
 
+  public function checkObsoleteHotfix() {
```

Comment:
Instead of this check, you can add to `extension-compatibility.json` and the extension will automatically be labelled "Obsolete" on the manage-extensions page.

Reply:
OK, cool, that's made unnecessary. Other comments about this are on the other subthread.